golang/cosmos/x/vbank/client/rest/query.go

```diff
@@ -0,0 +1,309 @@
+package rest
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/gorilla/mux"
+
+	"github.com/cosmos/cosmos-sdk/x/distribution/client/common"
+	"github.com/cosmos/cosmos-sdk/x/distribution/types"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/rest"
+)
+
+func registerQueryRoutes(clientCtx client.Context, r *mux.Router) {
```

Comment:
Are we intentionally squatting on the namespace normally used by `x/distribution`? If so, please write a prominent comment explaining the intention.

Reply:
You found the problem!  I accidentally copied and committed code that wasn't needed.